subsys/shell/shell.c

```diff
@@ -812,6 +812,19 @@ static int execute(const struct shell *sh)
 			&argv[cmd_with_handler_lvl], &help_entry);
 }
 
+static void toggle_logs_output(const struct shell *sh)
+{
+	if (IS_ENABLED(CONFIG_SHELL_LOG_BACKEND)) {
+		struct shell_log_backend_control_block *cb =
+			sh->log_backend->control_block;
+		if (cb->state == SHELL_LOG_BACKEND_ENABLED) {
+			cb->state = SHELL_LOG_BACKEND_DISABLED;
+		} else if (cb->state == SHELL_LOG_BACKEND_DISABLED) {
+			cb->state = SHELL_LOG_BACKEND_ENABLED;
```

Comment:
Is it sufficient to set the state? Shouldn't you use `z_shell_log_backend_enable`/`z_shell_log_backend_disable`?

Reply:
I'll let @jakub-uC decide on that.